tests/unit/test_utils.py

```diff
@@ -59,3 +59,7 @@ def test_ansibles_python(m, is_py2):
     m()
     cmd = ap_command_exists()
     assert is_ansibles_python_2(cmd) == is_py2
+
+@pytest.mark.parametrize("build_time, expected", [("00:00:10", "10 seconds"), ("00:10:10", "10 minutes"), ("10:10:10", "10 hours"), ("01:01:01", "01 hour"), ("00:01:01", "01 minute"), ("00:00:01", "01 second")])
+def test_fancy_time(build_time, expected):
+    assert fancy_time(build_time) == expected
```

Comment:
you also need to import fancy_time the same way you used it in cli.py

Reply:
Done !